bridge-sdk/connectors/eth-connector/src/eth_connector.rs

```diff
@@ -60,21 +61,29 @@ impl EthConnector {
             .value(amount);
 
         let tx = call.send().await?;
+
+        tracing::info!(transaction_id = format!("{:?}", tx.tx_hash()), "Sent deposit transaction");
+
         Ok(tx.tx_hash())
     }
 
     /// Transfers ETH to the EthCustodian and sets recipient as an Aurora EVM account. A proof from this transaction is then used to mint nETH on Aurora
+    #[tracing::instrument(skip_all, name = "deposit to evm")]
     pub async fn deposit_to_evm(&self, amount: u128, recipient_address: String) -> Result<TxHash> {
         let eth_custodian = self.eth_custodian()?;
         let call = eth_custodian
             .deposit_to_evm(recipient_address, U256::zero())
             .value(amount);
 
         let tx = call.send().await?;
+
+        tracing::info!(transaction_id = format!("{:?}", tx.tx_hash()), "Sent deposit transaction");
```

Comment:
Also, it will be nice to provide link to the explorer. Otherwise the user need to think about which blockchain and which  network

Reply:
I think it's better to keep logs simple. transaction_id can be used for filtering in case we start publishing these logs somewhere. Also it would require passing the information about blockchain explorer to connectors which I'm not a fan of